### ISSUE TITLE
inspect should allow copying a style to clipboard

### ISSUE THREAD

**Issue by id0Sch** [GoogleChromeLabs/ProjectVisBug#232]
Hi,
It would be really useful to add a copy style / css / jss to the clipboard straight from the inspector tooltip.

Since the tooltip will also show all modifications done by the extension, allowing us to copy the style would be a huge productivity boost!

I'm referring to this tooltip: 
![image](https://user-images.githubusercontent.com/5864772/49084122-3e885a00-f257-11e8-8338-1eec685d4189.png)

Thanks!!

**Comment by argyleink** [comment]
agreed! join the convo here #54 where we're chattin about your described functionality and more 👍

**Comment by janfehlis** [comment]
> Hi,
> It would be really useful to add a copy style / css / jss to the clipboard straight from the inspector tooltip.
> 
> Since the tooltip will also show all modifications done by the extension, allowing us to copy the style would be a huge productivity boost!
> 
> I'm referring to this tooltip:
> ![image](https://user-images.githubusercontent.com/5864772/49084122-3e885a00-f257-11e8-8338-1eec685d4189.png)
> 
> Thanks!!

Agreed! 💯 One of the first things I was looking for. 😃

**Comment by janfehlis** [comment]
> agreed! join the convo here #54 where we're chattin about your described functionality and more 👍

I think, this is not what @id0Sch (and I 😉 ) mean.

**Comment by argyleink** [comment]
sorry if i misunderstood! i believe what you want then is/has been available? you can "pin" a tooltip by holding alt/opt and clicking, then highlight and copy whatever you need, including image urls. 

![github-copy-styles](https://user-images.githubusercontent.com/1134620/56323263-2474e880-6120-11e9-94f0-ead322d2801c.gif)

or is this a feature request similar to firefox's new feature where they have a button that puts the changed styles into the clipboard for you?

**Comment by argyleink** [comment]
this feature is being worked on right now https://github.com/GoogleChromeLabs/ProjectVisBug/pull/354 but it doesnt include the ability to grab local changes with a hotkey. lemme think about that a little bit!
